com.unity.netcode.gameobjects/Editor/NetworkTransformEditor.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace Unity.Netcode.Editor
 {
-    [CustomEditor(typeof(NetworkTransform))]
+    [CustomEditor(typeof(NetworkTransform), true)]
```

Comment:
This is all you should need to support ClientNetworkTransform in the editor.  I've tested it